madmin/api/modules/ftr_walkerarea.py

```diff
@@ -43,7 +43,7 @@ class APIWalkerArea(apiHandler.ResourceHandler):
                 "settings": {
                     "type": "text",
                     "require": False,
-                    "empty": None,
+                    "empty": "",
                     "description": "Number of walkers than can be in the area",
                     "lockonedit": False,
                     "expected": int

```

Comment:
I'm pretty sure this needs to be changed to a `str` or the int needs to be converted to string type when writing to disk.